src/main/java/org/deepsymmetry/beatlink/data/OpusProvider.java

```diff
@@ -161,10 +161,7 @@ public synchronized void attachMetadataArchive(File archive, CdjStatus.TrackSour
      */
     @API(status = API.Status.STABLE)
     public Database findDatabase(DataReference reference) {
-        if (reference.player >= 9 && reference.player <= 12) {  // This is an Opus Quad deck.
```

Comment:
We emulate 1-4 now, this is not needed.

Reply:
Agreed. But we should still check that `VirtualRekordbox` is running?

Reply:
Im not sure its required as it is just reacting to events being passed to MetadataFinder from VirtualRekordbox/VirtualCdj. Happy to set it up though if you feel it needs it.

Reply:
The reason I think it might is that IIRC this is permanently registered with the `MetadataFinder`, and so will be called always when metadata is being looked for. I am think of the case where someone keeps BLT running, and switches back and forth between connecting to real DJ Link gear, and an Opus Quad. I want to make sure OpusProvider doesn’t try to offer metadata when it shouldn’t. (Maybe the user forgot to detach metadata archives when switching between devices.)

Reply:
Good point, adding it in. Thanks for the explanation!